integration/helper/routeHelper.ts

```diff
@@ -0,0 +1,66 @@
+import request from 'supertest'
+import express from 'express'
+
+export async function getOutOfBoundsReadAarch64(
+  app: express.Express,
+  password: any,
+  firstOffset: any,
+  secondOffset: any
+) {
+  return request(app)
+    .get(`/scenario/out-of-bounds-readV2-aarch64?params=${password}&params=${firstOffset}&params=${secondOffset}`)
+    .set('Accept', 'application/json')
+    .set('Content-Type', 'application/json')
+    .send()
+    .then((response: any) => {
```

Comment:
I think the `any`s on response and err can be removed

Reply:
Yeah, weirdly when I first wrote it it enforced types on it, strange